schema/xsd/direct.xsd

```diff
@@ -59,6 +59,7 @@
                 </xs:element>
             </xs:choice>
             <xs:element name="signature-type" type="signature-type" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="on-behalf-of" type="signing-on-behalf-of" minOccurs="0" maxOccurs="1"/>
```

Comment:
Hvis en offentlig avsender angir `notifications` kan de kun signere på vegne av `SELF`. Angis `notifications-using-lookup` kan de kun signere på vegne av`OTHER`. Angis ikke `on-behalf-of` setter vi bare den eneste lovlige verdien, mens om en ulovlig kombinasjon angis valideres det i `NotificationInfo` i signering-backend.